### ISSUE TITLE
isle2: there should be another turnstile or gate for exit

### ISSUE THREAD

**Issue by Yuccaa** [loiste-interactive/infra-issues#1095]
They're supposed to be one way.

**Comment by sebastiannielsen** [comment]
You mean in the castle?

Dual direction turnstiles are pretty common in access Control schemes.
Here you can see a turnstile that is being set in "Free exit, Reader access required for entry".

(The mode of access is set with a keylock in the top part)
https://www.youtube.com/watch?v=VhfMZ_OSM6c

But there is also a lot of turnstiles that can be set in this way permanently, normally with a screw under a locked cover. One example of such a turnstile is the guard turnstile at a amusement park in sweden, to enter you need either a permission from the guard or scan a personell card, but its also a free exit turnstile for those that entered with a ticket at the main gates but wants to leave.

**Comment by Yuccaa** [comment]
Yeah I know those exist, but at the moment the gate is a bit confusing, especially because of the wonky physics.

**Comment by sebastiannielsen** [comment]
I think the wonky physics can be fixed by reducing the number of leafs (there exist 3 leaf and 2 leaf turnstiles, but for example 2 leaf the fencing must cover 180 degrees) or making the turnstile slightly larger. Play around with it.
